Tests/ViewInspectorTests/SwiftUI/ViewThatFitsTests.swift

```diff
@@ -0,0 +1,26 @@
+import XCTest
+import SwiftUI
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+final class ViewThatFitsTests: XCTestCase {
+    func testAllEnclosedChildTextViews() throws {
+        let sut = TestViewThatFits()
+        let longText = try sut.inspect().find(text: "Very long text that will only get picked if there is available space.")
+        let shortText = try sut.inspect().find(text: "Short text")
+
+        XCTAssertEqual(try longText.string(), "Very long text that will only get picked if there is available space.")
+        XCTAssertEqual(try shortText.string(), "Short text")
```

Comment:
The test intentionally validates that both children Views are inspectable even though in a real runtime scenario only one of them would be provided.